### PR TITLE
new logo in different sizes and colors

### PR DESCRIPTION
**Pull request by ralfharing**
Remove old site logo and logo used on social media profiles. Replace with new logo in various sizes and colors.

### CODE REVIEW

File: templates/gcd/bits/front_page_menu.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <div id="front_page_menu">
   <div id="front_page_menu_header">
-    <img src="{{ MEDIA_URL }}/img/logo_new.png" alt="GCD">
```
Comment (handrews) [inline templates/gcd/bits/front_page_menu.html]: Testing- does each comment send an email?  And who gets that email anyway?

**Comment by jochengcd** [comment]
As discussed on board, we will store the files with a TM added, but not without (at least for the large ones)

**Comment by ralfharing** [comment]
Ok ... so then you should merge this change, make the changes you want (adding the TM), and check those in as new versions. Same as any other thing in source control.

**Comment by jochengcd** [comment]
We had always the practise to only accept changes which were acceptable in their current form.

Anyway, this is resolved now, we got the new files with the TM, they are added to the gcd-support repo, since they don't have to do with the website.

**Pull request closed without merging**